internal/db/db.go

```diff
@@ -47,11 +50,15 @@ func Open(dbType DbType, connectionUrl string) (*gorm.DB, error) {
 }
 
 func GetGormLogFormatter(log hclog.Logger) func(values ...interface{}) (messages []interface{}) {
+	const op = "db.GetGormLogFormatter"
+	ctx := context.TODO()
 	return func(values ...interface{}) (messages []interface{}) {
 		if len(values) > 2 && values[0].(string) == "log" {
 			switch values[2].(type) {
 			case *pq.Error:
-				log.Trace("error from database adapter", "location", values[1], "error", values[2])
+				if log.IsTrace() {
```

Comment:
In the PR message you say:

> Note: the conversion from logging to events means that log levels are not longer a thing in Boundary and operators will need to use event filtering to control which events are emitted.

How does that impact this `log.IsTrace()`?

Reply:
`GetGormLogFormatter` is specically about specifying a logger for gorm and the only current single available is the level of logging in the logger.